pagopa/io-app
# chore: [IOBP-183] Missing non-experimental ds in payment method details

## Short description
added support for non-experimental DS in payment method details


<img width="250" alt="image" src="https://github.com/pagopa/io-app/assets/60693085/74b9c562-68fc-4e32-b252-c26bae6ed0c1">

## List of changes proposed in this pull request
- small refactor to remove `connect` syntax from touched component
- switch to experimental DS switch aware typography components
- experimental DS switch aware background and top bar color

## How to test
yarn jest, also navigate to various payment methods' detail screen and make sure that the pages correctly follow the design system both with the experimental flag on and off
